pkg/features/kube_features.go

```diff
@@ -275,6 +275,14 @@ const (
 	// deletion ordering.
 	HonorPVReclaimPolicy featuregate.Feature = "HonorPVReclaimPolicy"
 
+	// owner: @vaibhav2107
+	// beta: v1.32
```

Comment:
drop the version from the comment, the versioned file is a more accurate version of this